### PR TITLE
showcase: Deno Paste

### PR DESCRIPTION
**Pull request by stephenmelnicki**
Hello there! 👋 

I put together a little toy while learning Deno and Fresh. It's a simple text upload website that keep uploads around for a few hours before wiping the database. I thought others may find it helpful with their own projects, so I figured I'd see about sharing it.

Please let me know if there's anything you'd like for me to change or update. Thanks!

### CODE REVIEW

File: www/data/showcase.json [hunk 1]
```diff
@@ -229,6 +229,12 @@
     "github": "xijaja/fresh-notion-blog",
     "image": "fresh-notion-blog"
   },
+  {
+    "title": "Deno Paste",
+    "link": "https://deno-paste.deno.dev/",
+    "github": "stephenmelnicki/fresh",
```
Comment (kt3k) [inline www/data/showcase.json]: Is `stephenmelnicki/deno_paste` the right repository?

Comment (stephenmelnicki) [reply www/data/showcase.json]: Oh shoot! I must've missed this. I'll get this updated in just a moment.

Good catch! Thank you!

Comment (stephenmelnicki) [reply www/data/showcase.json]: And [that](https://github.com/denoland/fresh/pull/987/commits/b10fd25a539ec8131a45465a0c88970624bd5dc7) should do the trick.

Thanks again!

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**